### PR TITLE
Fixes #1005:Optimized app.

### LINKED ISSUE #1005: Reduce application size by using build optimizer .
**I'm submitting a ...**
- [ ] bug report
- [x] feature request

**Current behavior:**
Currently `--build-optimizer` flag is not used to build the project.It can be used to reduce application size.

**Expected behavior:**
Reduce application size by using build optimizer.

### PR DESCRIPTION
**Pull request by praveenojha33**
Fixes #1005 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] I have added necessary documentation (if appropriate)
- [x] Added Surge preview link

#### Changes proposed in this pull request:
Optimized the project's size.
For more refer to #1005 

- https://pr-1006-fossasia-susper.surge.sh

### CODE REVIEW

**Comment by praveenojha33** [comment]
@simsausaurabh Please review.

File: deploy.sh [hunk 1]
```diff
@@ -46,7 +46,7 @@ git checkout $SOURCE_BRANCH
 
 # Actual building and setup of current push or PR. Move to `TARGET_BRANCH` and move the `dist` folder
 npm install
-ng build --prod --aot
```
Comment (simsausaurabh) [inline deploy.sh]: Do not remove ```--aot```.

File: package.json [hunk 2]
```diff
@@ -9,8 +9,8 @@
     "test": "ng test --single-run --code-coverage --reporters=coverage-istanbul",
     "pree2e": "webdriver-manager update",
     "e2e": "protractor",
-    "postinstall": "ng build --prod --aot",
```
Comment (simsausaurabh) [inline package.json]: Do not remove ```--aot```.

File: package.json [hunk 3]
```diff
@@ -9,8 +9,8 @@
...
-    "deploy": "ng build --prod --aot --base-href=/susper.com/ && cp ./dist/index.html ./dist/404.html && ./node_modules/.bin/angular-cli-ghpages --no-silent"
```
Comment (simsausaurabh) [inline package.json]: Do not remove ```--aot```.

File: deploy.sh (hunk 1 above)
Comment (simsausaurabh) [inline deploy.sh]: Do not remove ```--aot```.

File: package.json (hunk 2 above)
Comment (simsausaurabh) [inline package.json]: Do not remove ```--aot```.

File: .travis.yml [hunk 4]
```diff
@@ -17,7 +17,7 @@ before_install:
 
 # Build a production build of the project
 before_script:
-  - ng build --prod
```
Comment (simsausaurabh) [inline .travis.yml]: Please add ```--aot```.

Comment (praveenojha33) [reply .travis.yml]: Actually there is no need of adding `--aot` I have gone through it and found that if we use production build then `--aot` is already used, so there is no need of specifying it. Have a look here https://angular.io/guide/aot-compiler

Comment (praveenojha33) [reply .travis.yml]: I have also found that there is no difference in compiled files by using `ng build --prod` and `ng build --prod --aot`

**Comment by praveenojha33** [comment]
@mariobehling Please review.

**Pull request merged**